databases/db.js

```diff
@@ -22,7 +23,9 @@ const Promotion = PromotionsModel(sequelize, Sequelize); //Nueva implementacion
 const game = gamesModel(sequelize, Sequelize);
 const team = teamsModel(sequelize, Sequelize);
 const inventory = inventoryModel(sequelize, Sequelize);
-const Deposit = 
+//const Deposit =
+const money= moneyModel(sequelize,Sequelize);
```

Comment:
Mayusculas para los modelos pls
```suggestion
const Money= moneyModel(sequelize,Sequelize);
```